spec/models/amusement_park_spec.rb

```diff
@@ -3,5 +3,26 @@
 RSpec.describe AmusementPark, type: :model do
   describe "relationships" do
     it { should have_many(:rides) }
+    it { should have_many(:mechanics).through(:rides) }  
```

Comment:
Great work testing this additional association and model method! Keep in mind, we also want to see a feature spec for your amusement park show page to verify that the data requested in US 3 is rendering properly.

Reply:
I'm confident that given 3 hours, you would be able to get a feature spec set up! I missed the comment that said you did this in only 1 hour!